toolset/setup/linux/installer.py

```diff
@@ -19,6 +19,14 @@ def install_software(self):
     linux_install_root = self.fwroot + "/toolset/setup/linux"
     imode = self.benchmarker.install
 
+    script_vars = {
+      'TFB_DBHOST': self.benchmarker.database_host
+    }
+    l=[]
+    for k,v in script_vars.iteritems():
+      l.append(k+"="+v)
```

Comment:
I'd vote to use export e.g. `l.append("export %s=%s" % (k,v))` - there's not much security risk and it ensures that even if there's a future change this variable will always be available